sentry_streams/sentry_streams/examples/blq_fn.py

```diff
@@ -0,0 +1,35 @@
+import json
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+from enum import Enum
+
+
+class DownstreamBranch(Enum):
+    DELAYED = "delayed"
+    RECENT = "recent"
+
+
+@dataclass
+class KafkaMessage:
+    key: str
+    headers: dict[str, str]
```

Comment:
I don't think we 'd want to propagate these Kafka implementation details through the pipeline. Also we will not want to call the message class KafkaMessage.

Since this is an example we use it to showcase the api. 
Please rename it to Message and remove key and headers.